inc/customizer/loader.php

```diff
@@ -153,6 +154,19 @@ public function enqueue_customizer_controls() {
 						],
 					],
 					'deal'                          => ! defined( 'NEVE_PRO_VERSION' ) ? $offer->get_localized_data() : array(),
+					'starterContent'                => array(
+						'active'          => is_customize_preview() && current_theme_supports( 'starter-content' ) && get_option( 'fresh_site' ),
```

Comment:
Is there a reason we check for the customize preview & theme support here? This never loads outside the customizer and the theme supports starter content by default. 

I think this should be enough:

```suggestion
						'active'          => (bool) get_option( 'fresh_site' ),
```